usecases/example2/MessageRoutingSampleProject/K8sMessageRoutingSample/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM wso2/micro-integrator:latest
```

Comment:
@sajithaliyanage rather than referring to the `latest` image tag, can we refer to a specific version tag as it is the best practice when implementing Dockerfiles?

Reply:
Sone